<launch>
	<node name="sara_ui_raspberry"	pkg="sara_ui"	type="sara_ui" output="screen"/>
</launch>
